site/content/blog/2022-10-01-zap-and-hacktoberfest/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "ZAP and Hacktoberfest"
+summary: >
+    ZAP is partcipating in Hacktoberfest 2022.
+images:
+- https://www.zaproxy.org/site/blog/2022-10-01-zap-and-hacktoberfest/images/Hfest-Badge-2-Color-Void@2x.png
+type: post
+tags:
+- blog
+- hacktoberfest
+date: "2022-10-01"
+addSocialPreview: true
+authors:
+- arkaprabha
+---
+![Hacktoberfest X ZAP](images/zap_Hacktoberfest.png)
+
+We are happy to announce that ZAP is partcipating in Hacktoberfest 2022. Hacktoberfest 2022 is a month-long celebration of open source software presented by [DigitalOcean](https://digitalocean.com/), [Appwrite](https://appwrite.io/) and [Docker](https://docker.com/). Hacktoberfest is open to everyone in our community! You can sign up anytime between October 1 and October 31.
+
+## Contributors
+This is what contributors need to know to participate and complete Hacktoberfest 2022:
+- Register anytime between September 26 and October 31.
+- Contributions/pull requests can be made to any repositories under the [OWASP ZAP](https://github.com/zaproxy) organization.
```

Comment:
This seems to be an invitation to unwanted pull requests, not all repositories should be changed.

Reply:
Okay so just zaproxy, zap-extensions and zap-hud?

Reply:
I thought community scripts can be added too :)

Reply:
The ones linked from https://www.zaproxy.org/docs/developer/quick-start-build/ and Community Scripts sounds good.

Reply:
I'll add a bullet list

Reply:
added the list